app/components/HowCanIHelp/Career.tsx

```diff
@@ -0,0 +1,650 @@
+import Card from '../Card'
```

Comment:
you're going to have to move this to `app/routes/how-can-i-help.career.tsx`, which will be a bit of a problem - sorry about that

Reply:
All good, the change was needed. I've fixed it all now